accounts/models.py

```diff
@@ -319,7 +319,7 @@ class DemographicData(models.Model):
         ('rural', _('rural')),
     )
     user = models.ForeignKey(
-        User, on_delete=models.CASCADE,
+        User, on_delete=models.CASCADE, null=True,
```

Comment:
I added that because I couldn't figure out how to get the active user and set it on the form prior to saving the demographic data. So it gets saved once and then I add the user when I have it in scope and save it again. Not the way I wanted to do it, but it's done.